src/main/fc/config.c

```diff
@@ -202,8 +202,12 @@ static void validateAndFixConfig(void)
             pidProfilesMutable(i)->dyn_lpf_dterm_min_hz = 0;
         }
 
+        // Dterm lowpass 1 cutoff is not used when dynamic lowpass is enabled, but setting to 0 is the indicator
+        // to older configurators that lowpass 1 is disabled which results in resetting the type to PT1.
+        // This causes problems then because the dynamic lowpass still uses the type which has been incorrectly changed.
+        // So use the dyn_lpf_dterm_min_hz instead if it's enabled as a slightly less confusing (but unused) default.
         if (pidProfilesMutable(i)->dyn_lpf_dterm_min_hz > 0) {
-            pidProfilesMutable(i)->dterm_lowpass_hz = 0;
+            pidProfilesMutable(i)->dterm_lowpass_hz = pidProfilesMutable(i)->dyn_lpf_dterm_min_hz;
```

Comment:
This is wrong - we should not make it appear as if there is a lowpass at this frequency that simply does not exist. This will only confuse users, and require extra warnings to be added to configurator.

Reply:
Using 0 is not an option because of the configurator issue. Using the max is just as misleading because there is no extra lowpass at that frequency either and using the max puts the user in a precarious position if they disable the dynamic and switch back to static as the high cutoff would be completely inappropriate and lead to flyaways and/or hot motors.

The configurator already handles it well to only show the appropriate settings. With dynamic on:

![Screen Shot 2019-04-02 at 8 48 25 AM](https://user-images.githubusercontent.com/17088539/55403800-9f38e380-5524-11e9-90f7-c0baf472addc.png)

Static on:

![Screen Shot 2019-04-02 at 8 48 40 AM](https://user-images.githubusercontent.com/17088539/55403821-a65ff180-5524-11e9-9c10-4921d4e3499b.png)

Reply:
Ok, we'll just add a warning about the nonsensical configuration values shown in CLI to the configurator then.